package.json

```diff
@@ -93,26 +95,26 @@
     ]
   },
   "dependencies": {
-    "async": "^2.1.4",
-    "did-jwt": "0.0.8",
-    "ethjs-abi": "^0.1.8",
-    "ethjs-util": "^0.1.3",
-    "ipfs-mini": "^1.0.1",
+    "did-jwt": "^0.0.8",
+    "ethjs-abi": "^0.2.1",
+    "ethr-did-resolver": "^0.0.8",
     "mnid": "^0.1.1",
     "mobile-detect": "^1.3.5",
-    "nets": "^3.2.0",
-    "qr-image": "^3.1.0",
+    "pubsub-js": "^1.6.0",
     "qs": "^6.2.1",
-    "uport": "^0.7.0-alpha-minor-13",
-    "uport-core": "^0.0.43",
-    "web3": "0.19.0"
+    "store": "^2.0.12",
+    "uport": "^0.7.0-alpha-5",
+    "uport-core": "^0.0.48",
+    "uport-did-resolver": "^0.0.2",
+    "uport-lite": "^1.0.2"
```

Comment:
Do we need to include this directly? Doesn't it pick it up from the did resolver?